src/components/ClusterDeployment/ClusterDeploymentWizardFooter.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import Alerts from '../ui/Alerts';
+import { useAlerts } from '../AlertsContextProvider';
+import { WizardFooter } from '../ui';
+import { WizardFooterGenericProps } from '../ui/WizardFooter';
+import { Alert, AlertGroup, AlertVariant } from '@patternfly/react-core';
+import { CLUSTER_DEPLOYMENT_FIELD_LABELS } from './constants';
+
+type ClusterDeploymentWizardFooterProps = WizardFooterGenericProps & {
+  // additionalActions?: React.ReactNode;
+  errorFields?: string[];
+};
+
+const ValidationSection = ({ errorFields = [] }: { errorFields?: string[] }) => {
```

Comment:
why the default ?

I think the `ValidationSection` doesnt even have to render `AlertGroup` if there are no errorFields

Reply:
done. I keep the default but got your point and changed.